src/emuseum.js

```diff
@@ -13,33 +46,19 @@ module.exports = ({ emuseumKey }) => {
 
   return {
     async getObjects ({ ids }) {
-      const rawObjects = await this.getRawObjects({ ids })
+      let rawObjects;
```

Comment:
maybe run through `npm run format`

I wonder how easy it would be to add that as a hook to PRs...

Reply:
ah nice, yeah ill do that.